PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Install CrowdStrike Falcon Agent.md

```diff
@@ -0,0 +1,60 @@
+#### Name
+
+Windows - Install CrowdStrike Falcon Agent | v1.0 JCCG
+
+#### commandType
+
+windows
+
+#### Command
+
+```
+$CID="ENTER CID HERE"
+$installerURL="ENTER URL TO FALCON AGENT INSTALLER HERE"
+
+##### Do Not Edit Below This Line #####
+
+$installerTempLocation="C:\Windows\Temp\CSFalconAgentInstaller.exe"
+
+if(Get-Service "CSFalconService" -ErrorAction SilentlyContinue) {
+    Write-Host("Falcon Agent already installed, nothing to do.")
```

Comment:
You don't need the parentheses for cmdlet calls, they don't do anything here.

Reply:
fixed